### ISSUE TITLE
S0C4 on Zowe 1.19.1 with ZWED_agent_http_ipAddresses=0.0.0.0

### ISSUE THREAD

**Issue by jthyssenrocket** [zowe/zss#264]
If I add
ZWED_agent_http_ipAddresses=0.0.0.0
to my Zowe instance.env then zssServer dies with:

```
2021-03-11 13:48:02 <ZWELS:83952031> DOESTC INFO (start-component.sh:74) starting component zss ...                      
zssServer.sh removing /proj/ndx/var/zowe/inst1/logs/zssServer-2021-02-19-03-08.log                                       
ZSS_LOG_FILE=/proj/ndx/var/zowe/inst1/logs/zssServer-2021-03-11-08-48.log                                                
CEE3204S The system detected a protection exception (System Completion Code=0C4).                                        
         From entry point reportSLHFailure at statement 1447 at compile unit offset +000000BC at entry offset +000000BC  
         at address 209BA924.                                                                                            
ZWES1013I ZSS Server has started. Version '1.19.0+20210106'                                                              
ZWES1033I Using server config file '/proj/ndx/var/zowe/inst1/workspace/app-server/serverConfig/server.json'              
could not get any memory for SLH extension, extension size was 0x10               
```                    

As soon as I remove ZWED_agent_http_ipAddresses then zssServer starts fine

**Comment by 1000TurquoisePogs** [comment]
The env var config is correct, but the crash is unexpected. However, I tried this same thing on 1.19 and it worked fine for me, so it seems that there is an inconsistent memory issue that some users will experience but not others. we'll look into it more.

**Comment by 1000TurquoisePogs** [comment]
It seems this got fixed in 1.21 if not earlier, and was caused by a limitation in environment variable reading. I recall the team checking that a fix worked for @jthyssenrocket . If it still appears in 1.21, we can reopen and continue investigation.
